stdlib/public/core/MutableCollection.swift

```diff
@@ -269,6 +283,46 @@ extension MutableCollection {
   }
 }
 
+@available(SwiftStdlib 9999, *)
```

Comment:
I don't think this will work yet -- unless I'm mistaken, only 5.5 is defined in the CMake configs and the necessary build settings don't currently apply to the core stdlib.

In any case, I think we want to make these declarations backdeployable, in which case we don't want to declare availability for them.

Reply:
I'm a bit surprised referring to an unknown platform / version here doesn't emit an error diagnostic -- @xymus, is that by design? 

(If so, that may actually come useful in Swift System. 🤔)

Reply:
Yeah, that one was a mystery. I'll just take it out and see what happens!

Reply:
I was puzzled too so I checked. The compiler raises an error on a one word unknown availability macro, but only a warning on an unknown macro with a version number. This is because in case the macro doesn't match anything the compiler reverts to non-macro errors. We could clarify this in the future and adjust the diagnostic level at the same time.

You could use it in system until we have proper support in SwiftPM but that would likely mean a whole lot of warnings.

Reply:
Thanks